cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -62,12 +66,14 @@ func (pe *ProbeEvent) addBlockDeviceEvent(msg controller.EventMessage) {
 		klog.Infof("Processed details for %s", device.DevPath)
 
 		if isGPTBasedUUIDEnabled {
+			if isParentOrSlaveDevice(*device, erroredDevices) {
+				continue
```

Comment:
log an error for this device as well.

Reply:
a warning message that we skipped it because the device parent or child is already in an errored state.

Reply:
Done.